examples/gno.land/r/gnoland/pages/page_about.gno

```diff
@@ -4,17 +4,26 @@ func init() {
 	path := "about"
 	title := "Gno.land Is A Platform To Write Smart Contracts In Gno"
 	// XXX: description := "On Gno.land, developers write smart contracts and other blockchain apps using Gno without learning a language that’s exclusive to a single ecosystem."
-	body := `# About Gno.land
+	body := `
+Gno.land is a next-generation smart contract platform using Gno, an interpreted version of the general-purpose Go
+programming language. On Gno.land, smart contracts can be uploaded on-chain only by publishing their full source code,
+making it trivial to verify the contract or fork it into an improved version. With a system to publish reusable code 
+libraries on-chain, Gno.land serves as the “GitHub” of the ecosystem, with realms built using fully transparent, 
+auditable code that anyone can inspect and reuse.
 
-Gno.land is a platform to write smart contracts in Gno.
-Using an interpreted version of the general-purpose programming language Go (Golang), developers can write smart contracts and other blockchain apps without having to learn a language that’s exclusive to a single ecosystem.
-Web2 developers can easily contribute to web3 and start building a more transparent, accountable world.
+Gno.land addresses many pressing issues in the blockchain space, starting with the ease of use and intuitiveness of
+smart contract platforms. Developers can write smart contracts without having to learn a new language that’s exclusive 
+to a single ecosystem or limited by design. Go developers can easily port their existing web apps to Gno.land or build
+new ones from scratch, making web3 vastly more accessible.
 
-The Gno transaction token, GNOT, and the contributor memberships power the platform, which runs on a variation of Proof of Stake.
-Proof of Contribution rewards contributors from technical and non-technical backgrounds, fairly and for life with GNOT.
-This consensus mechanism also achieves higher security with fewer validators, optimizing resources for a greener, more sustainable, and enduring blockchain ecosystem.
+Secured by Proof of Contribution (PoC), a DAO-managed Proof-of-Authority consensus mechanism, Gno.land prioritizes 
+fairness and merit, rewarding the people most active on the platform. PoC restructures the financial incentives that 
+often corrupt blockchain projects and rewards contributors for their work based on their expertise, commitment, and 
```

Comment:
```suggestion
often corrupt blockchain projects, opting instead to reward contributors for their work based on expertise, commitment, and 
```

The current phrasing sounds to me like "what corrupts blockchain projects -> rewarding good contributors"